src/sentry/tasks/post_process.py

```diff
@@ -409,26 +409,20 @@ def post_process_group(
                 "is_reprocessed": is_reprocessed,
                 "has_reappeared": not group_state["is_new"],
             }
+            run_post_process_job(job)
 
-            _capture_group_stats(job)
-            process_snoozes(job)
-            process_inbox_adds(job)
-            handle_owner_assignment(job)
-            process_rules(job)
-            process_commits(job)
-            process_service_hooks(job)
-            process_resource_change_bounds(job)
-            process_plugins(job)
-            process_similarity(job)
-            update_existing_attachments(job)
-
-            if not is_reprocessed:
-                event_processed.send_robust(
-                    sender=post_process_group,
-                    project=event.project,
-                    event=event,
-                    primary_hash=kwargs.get("primary_hash"),
```

Comment:
`primary_hash` wasn't actually used in any of the receivers, so I just removed it to simplify things.